demo/api/src/news/generated/news.crud.resolver.ts

```diff
@@ -61,7 +65,7 @@ export class NewsCrudResolver {
             scope,
         });
 
-        await this.repository.persistAndFlush(news);
+        await this.entityManager.flush();
```

Comment:
Does this work without persisting the entity?

Reply:
if created via repository.create or em.create yes

https://mikro-orm.io/docs/5.6/configuration#persist-created-entities-automatically

Reply:
Didn't know this, thanks!